en/docs/assets/img/97566182/97566189/features/org.wso2.carbon.healthcheck.server_1.0.0/META-INF/DEPENDENCIES

```diff
@@ -0,0 +1,235 @@
+// ------------------------------------------------------------------
+// Transitive dependencies of this project determined from the
+// maven pom organized by organization.
+// ------------------------------------------------------------------
+
+WSO2 Carbon - Health Check API feature
+
+
+From: 'an unknown organization'
+  - icu4j  com.ibm.icu:icu4j:jar:2.6.1
+
+  - CLI  commons-cli:commons-cli:jar:1.0
+
+  - Codec  commons-codec:commons-codec:jar:1.2
+
+  - commons.dbcp.wso2 (http://wso2.org) commons-dbcp.wso2:commons-dbcp:bundle:1.4.0.wso2v1
+
+  - commons.io.wso2 (http://wso2.org) commons-io.wso2:commons-io:bundle:2.4.0.wso2v1
+
+  - Lang  commons-lang:commons-lang:jar:1.0
+
+  - commons.lang.wso2 (http://wso2.org) commons-lang.wso2:commons-lang:bundle:2.6.0.wso2v1
+
+  - commons.pool.wso2 (http://wso2.org) commons-pool.wso2:commons-pool:bundle:1.5.6.wso2v1
+
+  - Streaming API for XML  javax.xml.stream:stax-api:jar:1.0-2
+    License: GNU General Public Library  (http://www.gnu.org/licenses/gpl.txt)    License: COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0  (http://www.sun.com/cddl/cddl.html)
```

Comment:
Shall we move "License: COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0  (http://www.sun.com/cddl/cddl.html)" next line?